test/jit/test_save_load.py

```diff
@@ -70,6 +70,394 @@ def current_subcmul(a, b, alpha=1):
         self.assertEqual(no_alpha_current, current_subcmul(a, b))
         self.assertEqual(with_alpha_current, current_subcmul(*args))
 
+    # Helper that returns the module after saving and loading
+    def _save_load_module(self, m):
+        scripted_module = torch.jit.script(m())
+        buffer = io.BytesIO()
+        torch.jit.save(scripted_module, buffer)
+        buffer.seek(0)
+        return torch.jit.load(buffer)
+
+    # Helper which returns the result of a function or the exception the
+    #   function threw.
+    def _try_fn(self, fn, *args, **kwargs):
+        result = None
+        try:
+            result = fn(*args, **kwargs)
+        except Exception as e:
+            result = e
+        return result
+
+    def _verify_no(self, kind, m):
+        node_count = sum(kind in str(n) for n in m.graph.nodes())
+        self.assertEqual(node_count, 0)
+
+    def _verify_count(self, kind, m, count):
+        node_count = sum(kind in str(n) for n in m.graph.nodes())
+        self.assertEqual(node_count, count)
+
+    """
+    Tests that verify Torchscript remaps aten::div(_) from versions 0-3
+    to call either aten::true_divide(_), if an input is a float type,
+    or aten::floor_divide(_) otherwise.
+
+    NOTE: currently compares against current div behavior, too, since
+      div behavior has not yet been updated.
+    """
+
+    def test_versioned_div_tensor(self):
+        def historic_div(self, other):
+            if self.is_floating_point() or other.is_floating_point():
+                return self.true_divide(other)
+            return self.floor_divide(other)
+
+        # Tensor x Tensor
+        class MyModule(torch.nn.Module):
+            def __init__(self):
+                super(MyModule, self).__init__()
+
+            def forward(self, a, b):
+                result_0 = a / b
+                result_1 = torch.div(a, b)
+                result_2 = a.div(b)
+
+                return result_0, result_1, result_2
+
+        # Loads historic module
+        v3_module = torch.jit.load(pytorch_test_dir + "/jit/fixtures/test_versioned_div_tensor_v3.pt")
```

Comment:
is there an easy way to see / understand what's in the fixture?

Reply:
It's a serialized module like any other, so you could load it and print the graph. Is that what you're asking?